### PR TITLE
feat: add minted time for kolour details page

### PR DESCRIPTION
**Pull request by sk-yagi**
Resolve: https://app.asana.com/0/0/1204296603625121/f
image

### CODE REVIEW

File: modules/kolours/kolour.ts [hunk 1]
```diff
@@ -63,10 +63,14 @@ export async function getGenesisKreationWithKolours(
     : null;
 }
 
+type MintedKolourEntryDbRow = Omit<MintedKolourEntry, "createdAt"> & {
+  createdAt: Date;
```
Comment (sk-kitsune) [inline modules/kolours/kolour.ts:67]: @sk-shishi 😢

Comment (sk-kitsune) [reply modules/kolours/kolour.ts:67]: IMO, the correct way (in the ideal world) should be:

```
function parse$MintedKolourEntryDbRow(obj?: any): MintedKolourEntryDbRow;

const rows = await sql<unknown[]>`;
const entries = rows.map(row => parse$MintedKolourEntryDbRow(row));
```

Comment (sk-kitsune) [reply modules/kolours/kolour.ts:67]: let's ignore this issue for now

**Review by sk-kitsune — DISMISSED** [review]
✅ **Approved**: no blocking issues

File: modules/kolours/kolour.ts (hunk 1 above)
Comment (sk-shishi) [reply modules/kolours/kolour.ts:67]: Please don't use unknown[] 🥲 (if you want, it returns any[] if you Omit the generic param).

In the ideal world, we should have the type declarations generated by the actual SQL - like this https://pgtyped.dev

File: modules/kolours/kolour.ts [hunk 2]
```diff
@@ -95,7 +100,14 @@ export async function getAllMintedKolours(
...
   `;
-  return rows.slice();
+  return rows
+    .map(({ createdAt, ...others }) => {
+      return {
+        createdAt: createdAt.valueOf(),
+        ...others,
+      };
+    })
+    .slice();
```
Comment (sk-shishi) [inline modules/kolours/kolour.ts:110]: Drop the .slice

File: modules/kolours/types/Kolours.ts [hunk 3]
```diff
@@ -86,4 +86,5 @@ export type MintedKolourEntry = {
   userAddress: string;
   fee: LovelaceAmount;
   expectedEarning: LovelaceAmount | null;
```
Comment (sk-shishi) [inline modules/kolours/types/Kolours.ts:88]: Fyi, it cannot be null

File: containers/PageKolourDetails/index.tsx [hunk 4]
```diff
@@ -126,9 +127,11 @@ export default function PageKolourDetails({ className, style, kolour }: Props) {
                   <Typography.Div color="ink80" content="Minted Time" />
                   <Typography.Div
                     fontWeight="semibold"
-                    content={}
+                    content={moment(mintedKolour.createdAt).format(
+                      "MM-DD-yyyy HH:mm:ss"
```
Comment (sk-kitsune) [inline containers/PageKolourDetails/index.tsx:131]: image

**Comment by sk-yagi** [comment]
image

**Review by sk-kitsune — APPROVED** [review]
✅ **Approved**: no blocking issues

**Review by sk-umiuma — APPROVED** [review]

**Pull request merged**